### PR TITLE
BUD-2383 Updates for team budget feature

### PR DESCRIPTION
**Pull request by aparnasharma285**
Budget is releasing new feature called team budgets in August end. These changes are for the same.

### CODE REVIEW

**Comment by jonikamelcher** [comment]
HI @aparnasharma285. Have these changes been through the CPA review? In order to release this we would need approval from them and a release note announcing the changes.

**Comment by aparnasharma285** [comment]
Hi @jonikamelcher no it hasn't been through CPA review. I am not sure about the process. Could you point me the right team to reach out to.

**Comment by jonikamelcher** [comment]
@aparnasharma285 Alexei Potiagalov should be able to help you with the review, and may connect you with Alyson Nawossa for the launch process.

File: src/api-reference/budget/v4.budget-header.md [hunk 1]
```diff
@@ -817,6 +1255,111 @@ Authorization: Bearer {token}
...
+      "currencyCode": "USD",
+      "annualBudget": 1550.00000000,
+      "budgetCategory": null,
+      "owner": {
+          "externalUserSyncGuid": "8000011190000003453",
+          "employeeUuid": "F06D36B8AA9046358B8778D3B2A14FCF",
+          "email": "meenapriya.meenakshisundaram@sap.com",
+          "externalUserCUUID": "8000011190000003453"
+      },
+      "budgetManagers": [],
```
Comment (alexeipo) [inline src/api-reference/budget/v4.budget-header.md]: If no values should be passed for these fields, then I recommend to change the example to remove the fields from the payload assuming the API works that way.

**Review by alexeipo — COMMENTED** [review]
The change adds a new attribute. Therefore, a review is optional.

File: src/api-reference/budget/v4.budget-header.md (hunk 1 above)
Comment (aparnasharma285) [reply src/api-reference/budget/v4.budget-header.md]: Changes done

**Review by anuvenkataraman — APPROVED** [review]

**Comment by aparnasharma285** [comment]
@jonikamelcher  Could you merge this PR please.

**Comment by jonikamelcher** [comment]
Hi @aparnasharma285. Before this can be merged or released, we will need to publish a release note for this detailing the new functionality. Please submit a release note to  DL Concur_DevCenter <DL_5BA96883ECB2111D8C00004C@global.corp.sap>. Thank you.

**Comment by jonikamelcher** [comment]
:shipit:

**Pull request merged**